polkadot/node/subsystem-bench/src/lib/configuration.rs

```diff
@@ -35,24 +35,39 @@ pub struct PeerLatency {
 	pub std_dev: f64,
 }
 
-// Default PoV size in KiB.
+// Based on Kusama `max_validators`
+fn default_n_validators() -> usize {
+	300
+}
+
+// Based on number of kusama validators producing blocks
```

Comment:
This is not related to the validators producing blocks. It's the total number of cores: cores assigned to specific paras plus on-demand (chain config value)

Reply:
But we don't want to fetch all cores. We'd want to represent a single node effort. Roughly this number has to be set to `n_cores * needed_approvals / n_validators.`

Reply:
`n_cores * needed_approvals / n_validators.`  only so for availability read, for write, we need to cover all cores as said above.